src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  decreaseHealth(amount) {
+    this.health -= amount;
+
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter(animal => animal !== this);
+
+      return `${this.name}`;
+    }
+  }
 }
 
+Animal.alive = [];
```

Comment:
Here I think we need to use the key word 'static' but if your solution works, I don't even know... Let mentors judge then

Reply:
i can use static only in class